lib/practice_exercises.rb

```diff
@@ -1,13 +1,35 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: o(n)
+# Space Complexity: o(1)
 def remove_duplicates(list)
-  raise NotImplementedError, "Not implemented yet"
+  #raise NotImplementedError, "Not implemented yet"
+  array = list.length
+  index_1 = 0
+  array.times do
+    if list[index_1] == list[index_1 + 1]
+      list.delete_at(index_1)
```

Comment:
Actually `delete_at` will remove the element at `index_1` by shifting all the subsequent elements left by one index.  So it's an O(n) method.  Since you have it nested in a loop running n times.  This is actually an O(n<sup>2</sup>) method.